src/utils/dueDate.js

```diff
@@ -48,3 +50,12 @@ export default function dueDate(
 
   return formatDueDate(Number(milliseconds.toString()));
 }
+
+export const firstPaymentDueDate = () => {
```

Comment:
hooks should only use used inside hooks, so I would change this to a hook in the `hooks` folder. `useFirstPaymentDueDate`